compiler/rustc_ast/src/ast.rs

```diff
@@ -3031,14 +3031,16 @@ mod size_asserts {
     use super::*;
     use rustc_data_structures::static_assert_size;
     // These are in alphabetical order, which is easy to maintain.
-    static_assert_size!(AssocItem, 160);
```

Comment:
Was removing this assert intentional? I figure you'd instead want to keep it, but cfg(not(bootstrap)) it? (Apologies if it was, I skimmed through the discussion and couldn't see a mention of it).

Same for ForeignItem below.

Reply:
Thanks for the catch. Inane error during rebasing; I'll put this back in.